integration-cli/docker_cli_daemon_not_experimental_test.go

```diff
@@ -0,0 +1,39 @@
+// +build daemon,!windows,!experimental
```

Comment:
Currently, such a mechanism doesnt exist. We could add something in a follow on PR.